tests/ui/lifetimes/temporary-lifetime-extension-tuple-ctor.rs

```diff
@@ -34,3 +34,6 @@ fn main() {
     let g = some(&temp()); //~ERROR temporary value dropped while borrowed
     println!("{a:?} {b:?} {c:?} {d:?} {e:?} {f:?} {g:?}");
 }
+
+// ferrocene-annotations: fls_spdmit5fy7el
```

Comment:
I don't see where `target_feature` is used here, is it perhaps not the annotation intended?